apps/portal/src/app/bridge/swap/page.mdx

```diff
@@ -39,308 +40,21 @@ Build a custom token swap interface that allows users to exchange tokens across
-            className="flex-1 p-3 border rounded-lg focus:ring-2 focus:ring-blue-500"
-          />
-          <select
-            value={fromToken}
-            onChange={(e) => setFromToken(e.target.value)}
-            className="p-3 border rounded-lg focus:ring-2 focus:ring-blue-500"
-          >
-            <option value="ETH">ETH</option>
-            <option value="USDC">USDC</option>
-            <option value="WETH">WETH</option>
-          </select>
-        </div>
-        <p className="text-sm text-gray-500 mt-1">Ethereum</p>
-      </div>
-
-      {/* Swap Arrow */}
-      <div className="flex justify-center mb-4">
-        <button className="p-2 rounded-full bg-gray-100 hover:bg-gray-200">
-          ↓
-        </button>
-      </div>
-
-      {/* To Token */}
-      <div className="mb-6">
-        <label className="block text-sm font-medium mb-2">To</label>
-        <div className="flex gap-2">
-          <input
-            type="text"
-            placeholder="0.0"
-            value={quote ? (Number(quote.destinationAmount) / 1e18).toFixed(6) : ""}
-            readOnly
-            className="flex-1 p-3 border rounded-lg bg-gray-50"
-          />
-          <select
-            value={toToken}
-            onChange={(e) => setToToken(e.target.value)}
-            className="p-3 border rounded-lg focus:ring-2 focus:ring-blue-500"
-          >
-            <option value="MATIC">MATIC</option>
-            <option value="USDC">USDC</option>
-            <option value="WETH">WETH</option>
-          </select>
-        </div>
-        <p className="text-sm text-gray-500 mt-1">Polygon</p>
-      </div>
-
-      {/* Quote Info */}
-      {quote && (
-        <div className="mb-4 p-3 bg-blue-50 rounded-lg">
-          <p className="text-sm">
-            <span className="font-medium">Rate:</span> 1 {fromToken} = {(Number(quote.destinationAmount) / Number(quote.originAmount)).toFixed(6)} {toToken}
-          </p>
-          <p className="text-sm">
-            <span className="font-medium">Fee:</span> {((Number(quote.originAmount) - Number(amount) * 1e18) / 1e18).toFixed(6)} {fromToken}
-          </p>
-          <p className="text-sm">
-            <span className="font-medium">Estimated time:</span> {Math.round(quote.estimatedExecutionTimeMs / 1000)}s
-          </p>
-        </div>
-      )}
-
-      {/* Swap Button */}
-      <button
-        onClick={executeSwap}
-        disabled={!quote || isLoading || !account}
-        className="w-full py-3 px-4 bg-blue-600 text-white rounded-lg font-medium hover:bg-blue-700 disabled:bg-gray-300 disabled:cursor-not-allowed"
-      >
-        {!account ? "Connect Wallet" : isLoading ? "Getting Quote..." : "Swap"}
-      </button>
-
-      {!account && (
-        <p className="text-center text-sm text-gray-500 mt-3">
-          Connect your wallet to start swapping
-        </p>
-      )}
-    </div>
-  );
+function Example() {
+  return <SwapWidget client={client} />;
```

Comment:
reminder to add an example with prefill as well

Reply:
added